_commands/os_x/brew.md

```diff
@@ -0,0 +1,109 @@
+---
+---
+
+brew
+--
+
+`brew`, or known by its longer name as Homebrew, bills itself as the "missing package manager for OS X". Here's an example of installing a package: 
+
+~~~
+$ brew install wget
+==> Downloading https://homebrew.bintray.com/bottles/wget-1.17.1.el_capitan.bottle.1.tar.gz
+Already downloaded: /Library/Caches/Homebrew/wget-1.17.1.el_capitan.bottle.1.tar.gz
+==> Pouring wget-1.17.1.el_capitan.bottle.1.tar.gz
+🍺  /usr/local/Cellar/wget/1.17.1: 9 files, 1.5M
+~~~
+
+<!--more--> 
+
+That's right - `brew` uses emojis as part of its standard output 🙌 
+
+Various Linux distributions include package managers for installing software out of the box (e.g. `apt-get` for Ubuntu), and `brew` is a third-party effort to replicate the same for OS X. 
+
+To install `brew`, the [official Homebrew website](http://brew.sh) directs you to run this command, which incidentally explains itself as it runs:
+
+~~~
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+~~~
+
+As an example of `brew` usage, here's installing [Node.js](https://nodejs.org/):
+
+### Useful Options / Examples
+
+#### `brew install [package-name]`
+~~~
+$ brew install wget
+==> Downloading https://homebrew.bintray.com/bottles/wget-1.17.1.el_capitan.bottle.1.tar.gz
+Already downloaded: /Library/Caches/Homebrew/wget-1.17.1.el_capitan.bottle.1.tar.gz
+==> Pouring wget-1.17.1.el_capitan.bottle.1.tar.gz
+🍺  /usr/local/Cellar/wget/1.17.1: 9 files, 1.5M
+~~~
+
+##### Break it down
+
+The `brew install [package-name]` command is the command for installing various `brew` packages. There is a [public list](https://github.com/Homebrew/homebrew/tree/master/Library/Formula) of available packages that `brew` pulls from. Each package has a "formula", or package definition, that `brew` can parse. Upon picking a formula, like `node`, `wget`, or `cowsay`, `brew` retrieves the formula and follows the instructions (e.g. downloading the appropriate files) to install that package.
+
+#### `brew remove [package-name]`
+~~~
+$ brew remove wget
+Uninstalling /usr/local/Cellar/wget/1.17.1... (9 files, 1.5M)
+~~~
+
+##### Break it down
+
+The `brew remove` command is the command for uninstalling various `brew` packages, given the name of a package, using the package's formula.
+
+#### `brew update`
+~~~
+$ brew update
+Updated Homebrew from 86fc508 to 5531fd9.
+Updated 1 tap (homebrew/dupes).
+==> New Formulae
+...
+==> Updated Formulae
+...
+==> Deleted Formulae
+...
+~~~
+##### Break it down
```

Comment:
<img width="990" alt="screen shot 2016-03-17 at 7 31 44 pm" src="https://cloud.githubusercontent.com/assets/14142637/13864331/ef63fef4-ec76-11e5-8b27-aa986f76b35d.png">

Fix the leading #'s